crates/nu-data/src/value.rs

```diff
@@ -195,9 +195,10 @@ pub fn compute_values(
                 let result = match operator {
                     Operator::Plus => {
                         // FIXME: Not sure if I could do something better with the Span.
-                        let y = Primitive::into_chrono_duration(rhs.clone(), Span::unknown())
-                            .expect("Could not convert nushell Duration into chrono Duration.");
-                        Ok(x.checked_add_signed(y).expect("Data overflow."))
+                        match Primitive::into_chrono_duration(rhs.clone(), Span::unknown()) {
+                            Ok(y) => Ok(x.checked_add_signed(y).expect("Data overflow.")),
```

Comment:
so this will still panic if there's a data overflow. I think we want to do another check on the result of `checked_add_signed` and if it overflows, to return a different Err rather than panic

Reply:
Sorry for the delay. I'll take a look at this tomorrow.

Reply:
I've found some more cases where it panics due to overflows. Will see if I can fix and push those as well.